evap/static/scss/components/_grid.scss

```
@@ -77,19 +77,17 @@
         "name semester responsible participants result"
         / auto    6rem       12rem         7rem  11rem;
 
-    @include media-breakpoint-down(md) {
+    @include media-breakpoint-down(lg) {
         grid:
-            "name semester  responsible responsible"
-            "name semester participants      result"
-            / auto    3rem         7rem       10rem;
+            "name semester responsible participants result"
+            / auto    3rem       7rem         10rem;
     }
 
-    @include media-breakpoint-down(sm) {
+    @include media-breakpoint-down(md) {
         grid:
-            "        name        name    semester"
-            " responsible responsible responsible"
-            "participants      result      result"
-            /         1fr         1fr        3rem;
+            "name semester  responsible responsible"
+            "name semester participants      result"
+            / auto    1fr         1fr       3rem;
```

```suggestion
            / auto     1fr          1fr        6rem;
```
3rem results in very small bars that are hardly useful (not your fault). what do you think about using this larger value?

---

This is true. I updated it (see last commit)